.github/workflows/semantic.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   check-semantic-version:
-    if: "!contains(github.event.head_commit.message, 'skipci')"
+    if: github.event.pull_request.draft == false
```

Comment:
This is nicer than the skipci. But I believe for this to work successfully you'll need to add an extra trigger, because it needs to run not just when the PR is updated but when a PR is made ready for review (ie the draft status changes). More info here: https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request_target

Reply:
Thanks, updated!